pkgs/tools/security/chipsec/default.nix

```diff
@@ -28,6 +29,7 @@ python3.pkgs.buildPythonApplication rec {
   nativeBuildInputs = [
     libelf
     nasm
+    pahole
```

Comment:
```suggestion
] ++ kernel.moduleBuildDependencies;
```

Reply:
Thanks! I was not aware that `kernel.moduleBuildDependencies` was preferable to `pahole`.